var/spack/repos/builtin/packages/cub/package.py

```diff
@@ -34,6 +33,8 @@ class Cub(Package):
     url      = "https://github.com/NVlabs/cub/archive/1.6.4.zip"
 
     version('1.6.4', '924fc12c0efb17264c3ad2d611ed1c51')
+    version('1.4.1', '74a36eb84e5b5f0bf54aa3df39f660b2')
 
     def install(self, spec, prefix):
-        copy_tree('cub', prefix.include)
+        mkdirp(prefix.include)
+        install_tree('cub', '{0}/{1}'.format(prefix.include, 'cub'))
```

Comment:
You should use `join_path(prefix.include, 'cub')`. `join_path` is basically an alias to `os.path.join` and is used commonly in Spack for this purpose.